mvtools/terraform-iac/ec2-userdata.tftpl

```diff
@@ -45,9 +45,11 @@ mkdir -p /root/code
 aws s3 cp s3://${s3_bucket}/code.zip /tmp/code.zip
 unzip /tmp/code.zip -d /root/code/
 aws s3 cp s3://${s3_bucket}/results.csv /root/code/results.csv 
-python3 /root/code/mvtools/python-survey-app/questionnaireresults.py
+python3 root/code/mvtools/python-survey-app/questionnaireresults.py > /root/code/clean_results.csv
+python3 questionnaireresults.py 2> /root/code/error.txt
+aws s3 cp /root/code/error.txt s3://${s3_bucket}/error.txt
 python3 /root/code/mvtools/python-survey-app/questionnaireresults.py > /root/code/results.txt
-aws s3 cp //root/code/results.txt s3://${s3_bucket}/results.txt
+aws s3 cp //root/code/mvtools/python-survey-app/cleanedresults.csv s3://${s3_bucket}/cleanedresults.csv
```

Comment:
It seems that you have 2 versions of cleaned results csv? 
1. clean_results.csv
2. cleanedresults.csv

It seems to be a typo. If it is, correct it, if it isn't - where is cleanedresults.csv coming from? And why do you save print output to csv?

Reply:
As I input a CSV, I outputed a CSV... I didn't realise it was supposed to be a txt file! (and it was a typo)